bin/ssl.sh

```diff
@@ -67,7 +67,7 @@ openssl req \
 # this generates a "certificate signing request" file
 openssl req \
   -new \
-  -subj "/O=testsuite/OU=Node.js agent team/CN=ssl.lvh.me" \
+  -subj "/O=testsuite/OU=Node.js agent team/CN=localhost" \
```

Comment:
learned the hard way, you have to delete self signed cert and ca-index if they already exist, which they prob do

```
rm test/lib/self-signed-test-certificate.crt test/lib/ca-index
```

Reply:
`./bin/ssl.sh clear` also does the trick.